wikimedia/wikipedia-ios
# Base survey prompt URLs and translations off of primary app language

**Phabricator:** 
https://phabricator.wikimedia.org/T322158

### Notes
This PR is to change the survey URL, survey prompt translations, and survey footer privacy policy link to be based on the user's primary app language, per task requirements.

For translations - I added them manually into the strings files in the Localizations subdirectory (and ran the import script so they copied into the iOS Native Localizations subdirectory), based on the spreadsheet mentioned in [this comment](https://phabricator.wikimedia.org/T322158#8421741). I also made those changes on translatewiki's side, so the next PR from them won't strip them out.

### Test Steps
1. Trigger survey prompt while various languages are the primary (see "Feedback Form Links" section in Phab task). Confirm they go to the correct privacy policy link and survey link. Confirm translations on the survey prompt are in the correct language. It may help to comment out the user defaults [check](https://github.com/wikimedia/wikipedia-ios/commit/0dabe51994250ee8a95d596664a60651c7606596#diff-e3c2665359c4b1974fe1ea867c37d077062051b92f0cdb08a9e44d1a1e5e183eR355) while testing.
2. Also trigger it with a primary language not in this list. Confirm it takes user to the English survey link and privacy policy, and displays strings in English.

### Screenshots
See https://phabricator.wikimedia.org/T322158#8426040.
